Guessing.js

```diff
@@ -0,0 +1,53 @@
+/*
+1. have lower and upper bound
+2. Guess the Number
+3. Ask higher or lower
+4. If correct: done, otherwise update bound, go back to step 1
+*/
+
+/*
+asyc wait is for input only
+if-else statements
+variable assignments
+if (guess <= 50) {
+  console.log("");
+} else {
+  console.log("denied");
+}
+use loops probably while loops to keep
+
+do I need variables for 100 guesses?
+I don't think so bc, I have asyc and while looping from 1-100
+I also have hugher and lower
+*/
+
+const readline = require('readline');
+const readlineInterface = readline.createInterface(process.stdin, process.stdout);
+
```

Comment:
If you want to use the ask function to get user input then we need to define it up here first. That definition is:
```js
function ask(questionText) {
  return new Promise((resolve, reject) => {
    readline.question(questionText, resolve);
  });
}
```